chronoline/chronoline.js

```diff
@@ -394,13 +398,47 @@ function Chronoline(domElement, events, options) {
         // DRAWING
         t.floatingSet = t.paper.set();
         t.sectionLabelSet = t.paper.set();
+
+        //attach background click events function
+        t.myCanvas.onclick = function (e) {
```

Comment:
could save the events by setting `backgroundClick` to null in the defaults, then only instantiate and run this block if the callback event actually exists. Same goes for binding events below